packages/manager/modules/telecom-dashboard/src/index.js

```diff
@@ -0,0 +1,73 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'ng-at-internet';
+import 'at-internet-ui-router-plugin';
+import '@ovh-ux/ng-uirouter-title';
+import core from '@ovh-ux/manager-core';
```

Comment:
```suggestion
import '@ovh-ux/manager-core';
```

and then inject the module by using `'ovhManagerCore'`